### ISSUE TITLE
A11y: Map icons do not show in WHCM

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/clean-and-green-philly#335]
## Describe the task
1. When in Windows High Contrast Mode / dark the map icons are hard to see
Note: I'm not sure how customizable the map icons are. If it's not working out, please let me know.

<img width="724" alt="Screenshot 2024-02-10 at 2 39 25 PM" src="https://github.com/CodeForPhilly/vacant-lots-proj/assets/7329799/3c4417e2-71c0-49b6-a14a-63ee7faa4522">

## Acceptance Criteria

Open the page in WHCM dark or simulate it using Devtools(see link below)
- [ ] The map icons will be easier to see / not gray. Please use `ButtonText` for the color. See: https://www.smashingmagazine.com/2022/03/windows-high-contrast-colors-mode-css-custom-properties/
## Additional context

* https://www.smashingmagazine.com/2022/06/guide-windows-high-contrast-mode/
* How to simulate WHCM in Chrome Devtools - https://devtoolstips.org/tips/en/emulate-forced-colors/

**Comment by nlebovits** [comment]
@bacitracin does this still need to be worked on? if so, we can bump it in the slack channel

**Comment by bacitracin** [comment]
@nlebovits Yeah this one is still an issue. It's pretty low priority though, so no need to bump

**Comment by bacitracin** [comment]
Closing as not going to fix
